### PR TITLE
Added instructive aria-label and expanded shortcuts

### LINKED ISSUE #73: Add role-label and an aria-label to the game canvas
Following #7: results came back in and the game needs to be narrated to screenreader. We should add `role="img"` and an `aria-label` so the game can be narrated.

### LINKED ISSUE #74: Allow gameplay without a mouse
Following #7: results came back in and the game would benefit from having have shortcuts everywhere, such as for folks who don't have a mouse. Right now it only has shortcuts for some actions (namely P to pause/play).

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
Fixes #73. Fixes #74.

There are three very interlinked groups of changes in this PR:

* Added an `aria-label` to the game canvas that gets created or updated in each of the game scenes
* Moved displayed text to an `assets/strings.js` file to deduplicate the text from the aria-label and display 
* Added shortcuts to the end scene similar to the start scene

### CODE REVIEW

File: src/scenes/end.js [hunk 1]
```diff
@@ -74,12 +92,21 @@ export default class EndScene extends Phaser.Scene {
...
         );
     }
+
+    launchBuildYourOwn() {
+        window.location.href = "https://www.codecademy.com/learn/learn-phaser";
+    }
+
+    launchGame() {
+        this.scene.start("GameScene");
+        this.scene.stop("StartScene");
```
Comment (LinKCoding) [inline src/scenes/end.js:110]: I think it's largely inconsequential, but this should be `this.scene.stop('EndScene')`.

Comment (JoshuaKGoldberg) [reply src/scenes/end.js:110]: Ha, copy pasta error on my part... nice catch!

Comment (LinKCoding) [reply src/scenes/end.js:110]: > I think it's largely inconsequential, but this should be `this.scene.stop('EndScene')`.

**Comment by LinKCoding** [comment]
LGTM!

**Pull request merged**